backend/app.ts

```diff
@@ -1,13 +1,13 @@
 const express = require('express')
 const bodyParser = require('body-parser')
 
-const storage = require('./storage/storage')
-const matchRepository = require('./repositories/match-repository')
-const userRepository = require('./repositories/user-repository')
+const storage = require('./storage/Storage')
```

Comment:
why the rename?

Reply:
It seems like this convention is used in https://github.com/salsita/performance-review. It was also used in the PR that I am breaking up. Do we have some common convention defined somewhere? Should I keep file names as they were before?

Reply:
I don't have a strong preference 🙂 Feel free to keep the files renamed as long as it makes sense to you (if @novellizator is ok with that of course).

Reply:
I'm fine with this